tools/Schrodinger_evolution_cross_correlation_coefficients.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
Python notebooks should not be in mrcpp

Reply:
This is questionable...

Reply:
Why? Please explain. 
1. It is definitely not required for mrcpp to work
2. It could at least be made into a python script (not notebook)
Python notebooks clutter the commit history with many unnecessary commits (notebooks metadata change every time the script is run).

Reply:
Agree with Luca here that notebooks should not be in MRCPP. I will add to Luca's list that they're truly horrible to code review, despite the integration with ReviewNB. Please make it into a script.

Reply:
I do not think there is any point to run this script more often than once per several years. You used maybe ones and then keep only as a documentation.

Reply:
The best option, again is a simple python script that does exactly the same stuff. The python notebooks should be kept separate fromm the main code because what both Roberto and me said. Under git they make a mess and they don't belong there.

Reply:
The notion `separate` should be properly defined. On the last meeting I was told to use tools folder, and now I am being told differently. On my personal laptop is not a solution. Can you easily find generators of filters and cross correlation coefficients? It has to be on git somewhere.

Reply:
No, you're being told that the notebooks do not belong in MRCPP. It's fine to create a `tools` folder, but please, convert the code in the notebook to a Python script. It's not about how easy it is _for you_ to run these things _now_, it's about how easy you can make it for _other people_  to re-run things in the _next years_.